INSTALLATION.md

```diff
@@ -84,3 +85,47 @@ In the file add the line
 Replace `<username>` with the output of `whoami` and `/path/to/nginx` with the output of `which nginx`. Further information is available via [StackOverflow](http://askubuntu.com/questions/159007/how-do-i-run-specific-sudo-commands-without-a-password).
 
 You will also need to `chown` the Nginx error log so it is accessible by your user. You can see the location of this log file when you run `nginx -V` as the `--error-log-path` configure argument's parameter.
+
+### Use Nginx with Docker
```

Comment:
We should probably note that this only works on Linux (maybe on Windows?), but not on MacOS due to the virtualization.

Reply:
Windows might be the same as MacOS, but seems you can select `process` level isolation as well. I'll mention this.